src/engine/Settings.cs

```diff
@@ -215,9 +215,9 @@ private Settings()
         CustomUsername.Value != null ?
             CustomUsername.Value :
             Environment.UserName;
+            
 
     public int CloudSimulationWidth => Constants.CLOUD_X_EXTENT / CloudResolution;
-
```

Comment:
All changes in this file should be reverted.

Reply:
@Paulpicotin Still unintended changes here, you may want to use `git checkout master src/engine/Settings.cs` to revert these.